BUILD.gn

```diff
@@ -44,6 +44,42 @@ declare_args() {
   audit_dev_deps = true
 }
 
+# This target serves as a canonical place to define all code that has to be
+# buildable in all platforms in a single build run.
+group("all") {
+  testonly = true
+  deps = [
+    "//brave",
+    "//chrome",
+  ]
+
+  if (!is_android && !is_ios) {
+    deps += [
+      "//chrome/test:browser_tests",
+      "//chrome/test:unit_tests",
+      "//components:components_unittests",
+      "//content/test:content_unittests",
+      "//net:net_unittests",
+    ]
+  }
+
+  if (!is_ios) {
+    deps += [
+      "//brave/test:brave_browser_tests",
+      "//brave/test:brave_installer_unittests",
```

Comment:
@cdesouza the only notice - 
```
      "//brave/test:brave_installer_unittests",
      "//brave/test:brave_network_audit_tests",
```
must not be included on Android, other things lgtm

Reply:
Fixed.

Reply:
thanks